src/main/java/io/cdap/e2e/utils/GcpClient.java

```diff
@@ -30,37 +30,24 @@
  */
 public class GcpClient {
 
-  public int countBqQuery(String table) throws IOException, InterruptedException {
-    String projectId = SeleniumHelper.readParameters("Project-ID");
-    String datasetName = SeleniumHelper.readParameters("Data-Set");
-    String selectQuery = "SELECT count(*) "
-      + " FROM `"
-      + projectId
-      + "."
-      + datasetName
-      + "."
-      + table
-      + "`";
+  public static int countBqQuery(String table) throws IOException, InterruptedException {
+    String projectId = SeleniumHelper.readParameters(ConstantsUtil.PROJECT_ID);
+    String datasetName = SeleniumHelper.readParameters(ConstantsUtil.DATASET);
+    String selectQuery = "SELECT count(*) " + " FROM `" + projectId + "." + datasetName + "." + table + "`";
     return executeQuery(selectQuery);
   }
 
   //Deleting the table
-  public void dropBqQuery(String table) throws IOException, InterruptedException {
-    String projectId = SeleniumHelper.readParameters("Project-ID");
-    String datasetName = SeleniumHelper.readParameters("Data-Set");
-    String dropQuery = "DROP TABLE `"
-      + projectId
-      + "."
-      + datasetName
-      + "."
-      + table
-      + "`";
+  public static void dropBqQuery(String table) throws IOException, InterruptedException {
+    String projectId = SeleniumHelper.readParameters(ConstantsUtil.PROJECT_ID);
+    String datasetName = SeleniumHelper.readParameters(ConstantsUtil.DATASET);
+    String dropQuery = "DROP TABLE `" + projectId + "." + datasetName + "." + table + "`";
     executeQuery(dropQuery);
   }
 
-  private int executeQuery(String query) throws InterruptedException, IOException {
-    BigQuery bigquery = BigQueryOptions.newBuilder().setProjectId(SeleniumHelper.readParameters("Project-ID"))
-      .build().getService();
+  private static int executeQuery(String query) throws InterruptedException, IOException {
+    BigQuery bigquery = BigQueryOptions.newBuilder().setProjectId(
```

Comment:
Instead of creating a bigquery client on every call, create it once and use it in every method.

Reply:
done